kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/model/BalanceUpdate.java

```diff
@@ -0,0 +1,24 @@
+package com.kin.ecosystem.data.model;
+
+import java.math.BigDecimal;
+
+public class BalanceUpdate {
+
+    private BigDecimal amount;
+
+    public BalanceUpdate() {
+        this.amount = new BigDecimal(0);
+    }
+
+    public BalanceUpdate(BigDecimal amount) {
+        this.amount = amount;
+    }
+
+    public void setAmount(BigDecimal amount) {
+        this.amount = amount;
+    }
+
+    public BigDecimal getAmount() {
```

Comment:
should we return BigDecimal or integer - as we are working with whole numbers - it will be easer for developers

Reply:
I have 2 questions here:
why not use the same `Balance` object from kin-core?
I think that both sdk should use the same convention (either BIgDecimal or integer)

Reply:
hmmm, we are going to use BigDecimal to support also in the future not only integers.
@yosriz  we are not using the Balance from the core, because if something related to the balance will change it won't affect our API.